src/Microsoft.DocAsCode.Build.JavaScriptReference/AutoMapper/ApiReferenceBuildOutputConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using AutoMapper;
+
+    public class ApiReferenceBuildOutputConverter : ITypeConverter<string, ApiReferenceBuildOutput>
+    {
+        private readonly string[] _supportedLanguages;
+
+        private readonly IReadOnlyDictionary<string, ApiReferenceBuildOutput> _references;
+
+        public ApiReferenceBuildOutputConverter(string[] supportedLanguages, IReadOnlyDictionary<string, ApiReferenceBuildOutput> references)
+        {
+            if (supportedLanguages == null)
+            {
+                throw new ArgumentNullException(nameof(supportedLanguages));
+            }
+            if (supportedLanguages.Length == 0)
+            {
+                throw new ArgumentException(nameof(supportedLanguages));
```

Comment:
yes... will fix it.

---
In reply to: [93900347](https://github.com/dotnet/docfx/pull/1061#discussion_r93900347) [](ancestors = 93900347)